properties/pkg/properties_test.go

```diff
@@ -664,13 +633,13 @@ func TestProperties_Sort(t *testing.T) {
 
 		p.Sort()
 
-		if !cmp.Equal(p, expected) {
-			t.Errorf("Mismatch in expected and returned Properties!\nExpected: %q\nGot: %q\n",
-				expected, p)
-		}
+		g.Expect(p.Keys()).Should(Equal(expected.Keys()))
+		g.Expect(p.Equal(expected)).Should(BeTrue())
```

Comment:
```suggestion
		g.Expect(p).Should(BeEquivalentTo(expected))
```
Not sure the keys() check is required, I think the deepEqual under the hood could in fact cover it.